src/components/Table/Table.types.tsx

```diff
@@ -0,0 +1,31 @@
+export interface IPropsData {
+  id: string;
+  name: string;
+  birthDate: string;
+  phone: string;
+  city: string;
+}
+
+
+export interface IProps {
+  data: Array<IPropsData>;
+}
+
+export interface IHeaderProps {
+  data: Array<IPropsData>;
+  onClickHandle: (e: any) => void;
+  sortedBy: string | null;
```

Comment:
You don't need to write `| null` because you can assign null to variable with `string` type